webui/cypress/integration/server-details.spec.js

```diff
@@ -60,30 +83,64 @@ describe('Server details', () => {
     cy.get('.meta-test__ReplicasetServerListItem__dropdown *')
       .contains('Server details').click();
 
+    cy.get('.meta-test__ServerDetailsModal button:contains(Select zone)').click();
+    cy.get('div').contains('You have no any zone,').should('not.exist');
+    cy.get('div').contains('Narnia');
+    cy.get('button:contains(Add new zone)').click();
+    cy.get('.ZoneAddModal input[name="uri"]').type('Moscow');
+    cy.get('.meta-test__ZoneAddSubmitBtn').click();
+    cy.get('.ZoneAddModal_error').find('span:contains(Timeout exceeded)');
+    cy.get('h2:contains(Add name of zone)').next().click();
+    cy.get('.ZoneAddModal').should('not.exist');
+    
     cy.get('.meta-test__ServerDetailsModal').contains('Server status is "dead"');
     cy.get('.meta-test__ServerDetailsModal').contains('instance_uuid').should('not.exist');
-
+    
     cy.get('.meta-test__ServerDetailsModal button').contains('Cartridge').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('Replication').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('Storage').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('Network').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('General').click();
+    cy.get('.meta-test__ServerDetailsModal button').contains('Issues 0');
+    cy.get('button:contains(Close)').click();
 
-    cy.task('tarantool', {code: `_G.cluster:server('dummy-2'):start()`});
-
+    cy.exec('kill -SIGCONT $(lsof -sTCP:LISTEN -i :8082 -t)', {failOnNonZeroExit: false});
+    cy.reload();
+    cy.get('.meta-test__LoginBtn');
+    cy.get('.ServerLabelsHighlightingArea').contains(':13302').closest('li')
+      .contains('healthy');
+    cy.get('li').contains('dummy-2').closest('li')
+      .find('.meta-test__ReplicasetServerListItem__dropdownBtn').click();
+    cy.get('.meta-test__ReplicasetServerListItem__dropdown *')
+      .contains('Server details').click();
+      
     cy.get('.meta-test__ServerDetailsModal').contains('healthy');
     cy.get('.meta-test__ServerDetailsModal').contains('instance_uuid');
 
-    cy.task('tarantool', {code: `
-      return _G.cluster:server('dummy-2').instance_uuid
-    `}).then((resp) => {
-      const uuid = resp[0];
-      cy.get('.meta-test__ServerDetailsModal').contains(uuid);
-
-    });
-
-    cy.get('.meta-test__ServerDetailsModal button').contains('Close').click();
-    cy.get('.ServerLabelsHighlightingArea').contains(':13302').closest('li')
-      .contains('healthy');
+    cy.get('.meta-test__ServerDetailsModal button:contains(Select zone)').click();
+    cy.get('button:contains(Add new zone)').click();
+    cy.get('.ZoneAddModal input[name="uri"]').type('Rostov');
+    cy.get('.meta-test__ZoneAddSubmitBtn').click();
+    cy.get('.ZoneAddModal_error').find('span:contains(Two-phase commit is locked)', {timeout:15000});
```

Comment:
These problems are not related to server details. It's better not to mix them and test them separately.